src/main/sensors/gyro.c

```diff
@@ -480,13 +482,15 @@ void gyroInitFilterLpf(gyroSensor_t *gyroSensor, uint8_t lpfHz)
                 pt1FilterInit(&gyroSensor->softLpfFilter.gyroFilterPt1State[axis], lpfHz, gyroDt);
             }
             break;
-        default:
+        default:;
```

Comment:
is this `;` a typo?

Reply:
No, it's empty statement. It's illegal to have label at end of block (compound statement)

But I'll rewrite it, possibly by mowing `break` after `#endif` ...

Reply:
Ah I see. I just hadn't seen it used like that. Thanks for clarifying!